packages/editor-ui/src/components/AssignmentCollection/AssignmentCollection.vue

```diff
@@ -133,19 +134,27 @@ function optionSelected(action: string) {
 		</n8n-input-label>
 		<div :class="$style.content">
 			<div :class="$style.assignments">
-				<div v-for="(assignment, index) of state.paramValue.assignments" :key="assignment.id">
-					<Assignment
-						:model-value="assignment"
-						:index="index"
-						:path="`${path}.${index}`"
-						:issues="getIssues(index)"
-						:class="$style.assignment"
-						:is-read-only="isReadOnly"
-						@update:model-value="(value) => onAssignmentUpdate(index, value)"
-						@remove="() => onAssignmentRemove(index)"
-					>
-					</Assignment>
-				</div>
+				<Draggable
+					:key="JSON.stringify(state.paramValue.assignments.map((a) => a.id))"
+					v-model="state.paramValue.assignments"
+					handle=".drag-handle"
```

Comment:
Is this used? Same for other places, I don't see styles for the `drag-handle` class